src/lib/sidebar/PipelineSchemeForm.svelte

```diff
@@ -25,17 +25,73 @@
     scheme = getArbitraryScheme($gjSchemeCollection);
     scheme.pipeline ||= {
       scheme_type: "",
-      status: "",
-      timescale: "",
+      scheme_timescale: { status: "", timescale: "" },
```

Comment:
Similar to the other file, I think it's safer to do these as separate ||= statements below to keep backwards compatibility. If scheme.pipeline.scheme_budget is missing, set these defaults.

(I'm getting errors when I click scheme details with existing GJ files and with new ones)